src/main.ts

```diff
@@ -104,7 +106,12 @@ async function syncSession () {
   // Auto authenticate user only if they are not trying a login or logout
   if (!isSigningIn() && !isSigningOut()) {
     // Initialize token service which will do a check-sso to initiate session
-    await KeycloakService.initializeToken(null).then(() => {}).catch(err => {
+    await KeycloakService.initializeToken(null).then(() => {
+      const token = ConfigHelper.getFromSession(SessionStorageKeys.KeyCloakToken)
+      if (!token) {
+        ConfigHelper.removeFromSession(SessionStorageKeys.CurrentAccount)
+      }
+    }).catch(err => {
```

Comment:
I think it's OK but you could be please add a comment describing what this is doing?

"When the token is initialized, ..."

"If there is no token in session storage then this means ..."

Reply:
Nice, thanks.